apps/site/src/app/(home)/sections/FAQ/FAQ.tsx

```diff
@@ -18,14 +20,40 @@ const FAQ = async () => {
 
 	return (
 		<section className={styles.container}>
-			<div className={styles.faq}>
-				<div className={styles["vertical-line"]} />
-				<Image src={star} alt="star" className={styles["star-lg"]} />
-				<Image src={star} alt="star" className={styles["star-sm"]} />
-				<Image src={eraser} alt="eraser" className={styles["eraser"]} />
-				<h2 className={styles.title}>FAQ</h2>
+			<AnteaterFloatie
+				src={leftAnteater}
+				alt="left anteater on floatie"
+				className={styles["left-anteater"]}
+			/>
+
+			<Container as="div" className="m-0">
+				<div
+					className={`${styles["accordion-border"]} ${styles["answer-body"]} ${styles["header-body"]}`}
+				>
+					<span className="h4">
+						<h2 className="visually-hidden">FAQ</h2>
+						<span className={styles["light-blue-text"]}>FAQ!</span> Here's
+						answers to our most commonly asked questions!
```

Comment:
Thought: the Figma has this text set to font weight of 600 which in my opinion looks better than a font weight of 700. This also applies to all the Accordion headers (FAQ Questions) as well.